hedera-node/hedera-app/src/main/java/com/hedera/node/app/fees/congestion/MonoMultiplierSources.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.fees.congestion;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.hedera.node.app.service.mono.fees.congestion.FeeMultiplierSource;
+import com.hedera.node.app.service.mono.fees.congestion.MultiplierSources;
+import com.hedera.node.app.service.mono.utils.accessors.SignedTxnAccessor;
+import com.hedera.node.app.service.mono.utils.accessors.TxnAccessor;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+import java.time.Instant;
+
+public class MonoMultiplierSources {
+    private final MultiplierSources delegate;
+
+    public MonoMultiplierSources(
+            final FeeMultiplierSource genericFeeMultiplier, final FeeMultiplierSource gasFeeMultiplier) {
```

Comment:
I think we should introduce `@GasPriceMultiplier` and `@GenericPriceMultiplier` as well as bindings in `FeesInjectionModule` exactly like in mono:
```
@Binds
@Singleton
@GenericPriceMultiplier
FeeMultiplierSource bindGenericFeeMultiplierSource(TxnRateFeeMultiplierSource txnRateFeeMultiplierSource);

@Binds
@Singleton
@GasPriceMultiplier
FeeMultiplierSource bindGasFeeMultiplierSource(GasPriceMultiplierSource gasPriceMultiplierSource);

```

Reply:
Oh, you have the same thing but without the annotations inside `ThrottleInjectionModule` 👍